src/wp-admin/site-editor.php

```diff
@@ -61,8 +61,6 @@ static function ( $classes ) {
 	'siteUrl'                   => site_url(),
 	'postsPerPage'              => get_option( 'posts_per_page' ),
 	'styles'                    => get_block_editor_theme_styles(),
-	'defaultTemplateTypes'      => $indexed_template_types,
-	'defaultTemplatePartAreas'  => get_allowed_block_template_part_areas(),
 	'supportsLayout'            => wp_theme_has_theme_json(),
 	'supportsTemplatePartsMode' => ! wp_is_block_theme() && current_theme_supports( 'block-template-parts' ),
 );
```

Comment:
Just noting that because of these two changes to site editor and editor initialization, we'll need to wait for the next package update to be able to commit this PR.

Reply:
Just a heads-up that [Gutenberg 19.8 has been released](https://make.wordpress.org/core/2024/12/04/whats-new-in-gutenberg-19-8-4-december/).